core/src/sim.c

```diff
@@ -210,24 +212,6 @@ void ignore_play(SimmedPlay *sp, int lock) {
 }
 
 int handle_potential_stopping_condition(Simmer *simmer) {
-  if (simmer->num_simmed_plays < 2) {
-    return 1; // should stop
-  }
-  if (simmer->iteration_count >
-      (MAX_STOPPING_ITERATION_CT +
-       (simmer->max_plies * PER_PLY_STOPPING_SCALING))) {
-    return 1;
-  }
-  int ignored_plays = 0;
-  for (int i = 0; i < simmer->num_simmed_plays; i++) {
-    if (simmer->simmed_plays[i]->ignore) {
-      ignored_plays++;
-    }
-  }
-  if (ignored_plays >= simmer->num_simmed_plays - 1) {
-    // There is only one unignored play; exit.
-    return 1;
-  }
```

Comment:
Removing all of these early will add a little bit of time because now we have to lock and sort the plays etc first instead of exiting right away. Maybe it's not that much time.

Reply:
I made a small update to ensure that the early conditions should never be true.